tailormap-components/projects/core/src/lib/user-interface/attributelist/attribute-list-button/attribute-list-button.component.ts

```diff
@@ -13,9 +13,15 @@ import { take } from 'rxjs/operators';
 })
 export class AttributeListButtonComponent implements OnInit {
 
+  public tooltip: string;
+
   @Input()
-  public set attributeListConfig(config: AttributeListConfig) {
+  public set attributeListConfig(configJsonString: string) {
+    const config: AttributeListConfig = JSON.parse(configJsonString);
     this.store$.dispatch(setAttributeListConfig({ config: { ...config } }));
```

Comment:
Is the spread-operator necessary? I thought JSON.parse already cloned it

Reply:
Yes, good catch, removed.